code/common/execas.q

```diff
@@ -1,4 +1,4 @@
 \d .pm
 
 execas:{[f;u]
- $[`.pm.requ ~ key `.pm.requ;.pm.requ[u;f]; value f]}
+ $[`.pm.requ ~ key `.pm.requ;.pm.requ[u;f];`.pm.valp ~ key `.pm.valp;.pm.valp f; value f]}
```

Comment:
My only worry about the change is that this now makes the readonly property dependent on having the permissions library installed which I thought we wanted to avoid